docs/containerd-2.0.md

```diff
@@ -0,0 +1,97 @@
+# containerd 2.0
+
+Please try out the release binaries and report any issues at <https://github.com/containerd/containerd/issues/>.
+
+## What's new
+
+### Adds top level configuration migration support
+
+This release adds containerd configuration migration support during daemon startup time. This feature means previous versions of containerd configuration will continue to work on future daemon versions; however, it is recommended to migrate to the latest version (`version = 2`) to avoid migrations and optimize daemon startup time. See [RELEASES.md](../RELEASES.md#daemon-configuration) for full details.
+
+### Adds plugin info to Introspection API
+
+This release adds `PluginInfo` API definition to the introspection service. (`/containerd.services.introspection.v1.Introspection/PluginInfo`)
+
+`PluginInfo` defines the API for returning information directory from a [containerd] plugin if the plugin supports it.
+
+### Adds Sandbox Controller Update API
+
+This release adds the `Update` API definition for the sandbox controller. (`/containerd.services.sandbox.v1.Controller/Update`)
+
+`Update` defines the API that sandbox plugins will implement to allow for modifications to an existing sandbox's attributes such as [sandbox] specification, runtime, extensions, and labels.
+
+### Adds OTEL environment variable configuration support for containerd's built-in tracing plugin
+
+This release adds support for the standard environment variables defined in the OpenTelemetry [specification](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/) to the built-in containerd tracing plugin.
+
+Implementation note: by default OTEL will try to connect to `https://localhost:4318` if no endpoint is set. In contrast, the containerd tracing OLTP plugin will be disabled if no endpoint is configured.
+
+i.e. both `OTEL_SDK_DISABLED` and one of either `OTEL_EXPORTER_OTLP_ENDPOINT` or `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` must be set in the containerd environment for the tracing plugin to be enabled.
+
+### Enables NRI by default
+
+This release enables NRI by default. To disallow NRI functionality, users will need to explicitly disable the common NRI plugin in containerd configuration. See [Disabling NRI Support in Containerd](../docs/NRI.md#disabling-nri-support-in-containerd) for details.
+
+### Adds Intel ISA-L's igzip support
+
+This release adds support for Intel ISA-L's igzip to the containerd client. If found, the containerd client will use igzip for gzip decompression. Similar to other external gzip implementations (e.g. pigz), users can disable igzip using the `CONTAINERD_DISABLE_IGZIP` variable in the containerd client process environment.
+
+## What's breaking
```

Comment:
In the next revision, I would like to get some warning blurbs from deprecated features here where applicable.